### DISCUSSION TITLE
What tags to bring back old sheet UI ?

### DISCUSSION THREAD

**Discussion by CySxL** [PoomSmart/YTABConfig#44] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/15531137/204792243-36a789c9-2087-4018-96c9-a9a21f662bae.jpeg)
^New UI 

I want to bring back the old filled-style sheet UI of the menu. So uYou's button won't get removed with the newer UI.

**Comment by CySxL** [comment]
Found it myself lol

**Reply by PoomSmart** [reply]
Which is?

**Reply by CySxL** [reply]
Narrowing them down atm.

**Comment by CySxL** [comment, marked as answer]
Answer to myself :

Change this from 1 to 0 ->
uiSystemsClientGlobalConfigIosEnableSheetsUxUpdates: 0
(Test on YouTube 17.46.4)


New/Current UI :
(lose uYou button but has a dismiss grabber)
![image](https://user-images.githubusercontent.com/15531137/204801391-09db6911-c304-4602-9093-e3a0fae25ab7.jpeg)


Old/Filled UI :
(has uYou and cancel button)
![image](https://user-images.githubusercontent.com/15531137/204801560-e49bf8cc-36cc-4827-8b86-91462ae9b718.jpeg)


Share with you guys : )
